applications/matter_bridge/snippets/onoff_plug/bridge.matter

```diff
@@ -721,7 +722,12 @@ cluster Binding = 30 {
       and enforce Access Control for the Node's endpoints and their associated
       cluster instances. */
 cluster AccessControl = 31 {
-  revision 2;
+  revision 3;
```

Comment:
probably check if the revision should be updated - based on the current v1.5 version I don;t see update to revision 3

Reply:
Yeah, also, I found that all the zap files have been generated incorrectly. I need to provide a fix in zap-generate.